pkg/pool-utils/contracts/test/MockVault.sol

```diff
@@ -204,4 +204,13 @@ contract MockVault is IPoolSwapStructs {
 
         emit PoolBalanceChanged(poolId, msg.sender, tokens, deltas, dueProtocolFeeAmounts);
     }
+
+    // Needed to support authorizer adapter entrypoint
```

Comment:
```suggestion
    // Needed to support authorizer adaptor entrypoint
```

Reply:
Having to speak British trips me up. Blimey!